### ISSUE TITLE
Bank Customers Prediction 

### ISSUE THREAD

**Issue by aryantiwari10** [prathimacode-hub/ML-ProjectKart#419]
Define You:

- [x] LGM-SOC'21 Participant 
- [ ] DevIncept Participant
- [x] Contributor

PROJECT TITLE : Bank Customers Prediction 
plz assign this project to me.

GOAL : To predict the customers who are withdrawing their account from the bank due to some loss and other issues.

DATASET : https://www.kaggle.com/santoshd3/bank-customers

WHAT I HAD DONE : In this project first I performed a exploratory data analysis on the Bank Customer dataset which includes of data cleaning , data manipulation, data preprocessing , data visualization and after that I did training and testing , then I did the feature scaling and used ANN for the prediction. While in the EDA part I have included different plots for the different visualizations of our dataset .

LIBRARIES:

PANDAS
NUMPY
MATPLOTLIB
SEABORN
SCIPY
SKLEARN
Keras
TensorFlow

Conclusion : We get to know why customers are withdrawing their account from bank.

**Comment by prathimacode-hub** [comment]
Issue assigned. @aryantiwari10
